Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStore.mm

```diff
@@ -606,7 +606,7 @@
             [layer setValue:@1 forKeyPath:WKCGDisplayListBifurcationEnabledKey];
         } else
             layer.opaque = m_isOpaque;
-        auto data = std::get<IPC::SharedBufferReference>(*m_displayListBufferHandle).unsafeBuffer()->createCFData();
+        auto data = std::get<IPC::SharedBufferCopy>(*m_displayListBufferHandle).unsafeBuffer()->createCFData();
```

Comment:
I had the resolve a conflict here with regards to `buffer()` vs `unsafeBuffer()` so please make sure I resolved it right.